client/deposits/list/index.tsx

```diff
@@ -261,11 +275,14 @@ export const DepositsList = (): JSX.Element => {
 						)
 					);
 
-					wcpayTracks.recordEvent( 'wcpay_deposits_download', {
-						exported_deposits: exportedDeposits,
-						total_deposits: exportedDeposits,
-						download_type: 'endpoint',
-					} );
+					wcpayTracks.recordEvent(
+						wcpayTracks.events.DEPOSITS_DOWNLOAD_CSV_CLICK,
+						{
+							exported_deposits: exportedDeposits,
+							total_deposits: exportedDeposits,
+							download_type: 'endpoint',
```

Comment:
Not sure why we have `download_type`; is this event fired elsewhere with other values?

Reply:
Ah I see, both scenarios are in this file (`endpoint` `browser`).